docs/themes/thxvscode/layouts/apis/aggPackage.html

```diff
@@ -3,22 +3,30 @@
 {{ end }}
 
 {{ define "main" }}
+
+<!--
+    This template generates a page enumerating a list of child package pages (as a table).
+    It requires the following parameters:
+    - title: Text to use as the `h1` of the page
+    - summary: Summary text to display beneath the title.
+    - version: The API docs version to which the package links should resolve.
+    - children: List of package names, whose API docs will be linked in the table.
+      Note: the logic in this template assumes that API docs exist for the specified package name.
+      If no such docs are found, it will simply skip rendering an entry for them.
+-->
+
 <div class="container body-content docs">
     <div class="row">
         <div class="col-md-2 docs-navbar-container"> {{ partial "apiNav.html" .}} </div>
         <div role="main" id="main-content" class="col-sm-9 col-md-8 body">
             <div>
                 <h1>{{ .Title }}</h1>
-
-                {{ $packagePages := where .Site.AllPages "Params.kind" "Package" }}
-                {{ $topPackage := index (where $packagePages "Params.package" $.Params.package) 0 }}
-
                 <div>
-                    {{ $topPackage.Params.Summary | markdownify }}
+                    {{ .Summary | markdownify }}
                 </div>
 
-                <h3>List of bundled packages</h3>
```

Comment:
This text was specific to the `fluid-framework` package use-case, and wasn't particularly meaningful. Now that we're using this template more generally, it has been removed.